### ISSUE TITLE
inconsistent Requires-Dist leads to ImportError with 0.25

### ISSUE THREAD

**Issue by cburroughs** [tensorflow/probability#1854]
5fa62e16b7859d38afb427e45ffbe03959526e75 set a runtime check with `  required_tensorflow_version = '2.18'`, but https://github.com/tensorflow/probability/blob/v0.25.0/setup.py#L52 still has the `Requires-Dist ` set to 2.16.

This will lead to an import error like:
```
E   ImportError: This version of TensorFlow Probability requires TensorFlow version >= 2.18; Detected an installation of version 2.16.2. Please upgrade TensorFlow to proceed.
```

which could be avoided if the `Requires-Dist ` matched.

**Comment by ColCarroll** [comment]
thank you! 

upgrading tensorflow manually should work for now, right?

**Comment by cburroughs** [comment]
Yes, a user can work around this by manually satisfying the dependency.
